06-walkthrough-1.Rmd

```diff
@@ -348,12 +654,59 @@ m_course <- lmer(percentage_earned ~ TimeSpent + (1|course_id), data = dat)
 summary(m_course)
```

Comment:
I might sound like a broken record, but here again I think that comments on each line would help explain what is happening. the (1|course_id) especially is confusing in light of how we learned the lm function, so could warrant a little extra explanation.

Reply:
Also could be good to explain what we are looking at in the summary call in 654

Reply:
thanks, this was helpful - added a lot of explanation here: 

As different as these (the parentheses and bar) are, they are used for a relatively straightforward purpose: to model the group (in this case, courses) in the data. With `lmer()`, these group terms are specified in parentheses - specifically, to the right of the bar. That is what the `|course_id` part means - it is telling lmer that courses are groups in the data. The left side of the bar tells lmer that what we want to be specified are varying intercepts for each group (1 is used to denote the intercept). That is basically it!

That is basically it, but there is potentially more to the story: in addition to the 1, variables which can be specified to have a different effect for each group can also be specified. These variables are referred to not as varying intercepts, but as varying slopes. We will not cover these in this walkthrough, but want you to be aware of them (we recommend the book  by West, Welch, and Galecki [2014] provide an excellent walkthrough on how to specify varying slopes using lmer).